src/makeRobot.js

```diff
@@ -38,7 +38,67 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name},`
+      + ` chip version: ${this.version},`
+      + ` wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    evacuate() {
+      this.coords.x += 1400;
+      this.coords.y += 500;
+    },
+
+    goRight(value = 1) {
+      if (value < 0) {
+        return this;
+      }
+      this.coords.x += value;
+
+      return this;
```

Comment:
`return this;` don't repeat your self, fix in all cases
```suggestion
      if (value > 0) {
        this.coords.x += value;
      }
    
      return this;
```

Reply:
> `return this;` don't repeat your self, fix in all cases

Agree, fixed )